hydjet.spec

```diff
@@ -0,0 +1,20 @@
+### RPM external hydjet 1.9.1
+
+Source: http://cern.ch/lokhtin/hydro/%{n}-%{realversion}.tar.gz
+
+BuildRequires: cmake
```

Comment:
please also add build dependency on `gmake` otherwise it will pick system gmake.

Reply:
Thanks for advice. But, does it matter?

Reply:
yes it does, some times we end up in env (e.g. docker containers with minimal package installation) where gmake is not available via system

Reply:
I mean does it matter gmake, make, or ninja tool is used?

Reply:
it depends,  if you have run cmake to use ninja then  you need to add dependency on ninja otheriwse default is `make` (which is available via our gmake.spec) and for that you need to add dependency on gmake.

Reply:
Thank You for Your explanation!